### DISCUSSION TITLE
What's the typical latency between a new rabbitmq release and the docker images being updated

### DISCUSSION THREAD

**Discussion by Stono** [docker-library/rabbitmq#750] [General]
As the title says really, would love to give 4.0.5 a go but there's no docker image for it.

Couldn't see anywhere explaining how and when they're published.  I (perhaps wrongly) assumed it'd be somewhat automatic?  So wondering if a process has broken?

Ta

**Comment by michaelklishin** [comment]
[See a change merged here that doesn't show up on Docker Hub yet?](https://github.com/docker-library/rabbitmq) is right there in the README.

For many patch releases the answer is "a few days", 3-4 or so.

If I were to guess, https://github.com/docker-library/rabbitmq/issues/745 and (not specific to the RabbitMQ image, resolved the same day) https://github.com/docker-library/rabbitmq/issues/747 can increase the latency for `4.0.5` because after https://github.com/docker-library/rabbitmq/issues/747, there must have been an internal post-mortem and could result in some time-consuming changes to reduce the risk of similar failures happening again. That's just my guess, though.

**Reply by LaurentGoderre** [reply]
That and the holiday period means slower turnaround.

**Comment by Stono** [comment]
Ah thank you, I completely missed that.
Apologies for the oversight. 

I can follow 4.0.5 here it seems https://github.com/docker-library/official-images/pull/18126 👍
